net/udpbroadcastrelay/src/opnsense/mvc/app/controllers/OPNsense/UDPBroadcastRelay/forms/dialogEdit.xml

```diff
@@ -43,6 +43,31 @@
       <type>checkbox</type>
       <help><![CDATA[Use the ID to set the TTL Value of the packet (Original method) rather than DSCP, default is not ticked]]></help>
    </field>
+   <field>
+      <id>udpbroadcastrelay.allowcidr</id>
+      <label>Allowed CIDRs</label>
+      <style>tokenize</style>
+      <type>select_multiple</type>
+      <allownew>true</allownew>
+      <help><![CDATA[Only allow packets from a range of IP source addresses]]></help>
+      <advanced>true</advanced>
```

Comment:
I see all of these are advanced. I don't mind, but it looks interesting releasing a 1.1 and not offering new features at first glance :)

Reply:
MSearch definitely should be advanced feature (given the specific sytax requirements and function).

Maybe Allow/Block CIDR could be non-advanced?